Duplicate Compile entries cause CS2002 warnings

## Description

When generated `.cs` files already exist on disk (from a previous build), the SDK glob picks them up as `Compile` items with relative paths. Fluidify's `AddFluidifyCompileItems` target then does `Compile Remove` / `Compile Include`, but the `Remove` doesn't match because `OutputPath` is computed as an absolute path (`%(RootDir)%(Directory)%(Filename)`) while the SDK uses relative paths.

This results in duplicate `Compile` entries and CS2002 warnings:

```
CSC : warning CS2002: Source file '.../MultiArray.cs' specified multiple times
```

## Root cause

In `Fluidify.targets`, `ComputeFluidifyOutputs` computes:

```xml
<OutputPath>%(RootDir)%(Directory)%(Filename)</OutputPath>
```

This produces an absolute path like `/home/user/project/src/MyProject/MultiArray.cs`.

The SDK glob adds the same file as a relative path like `MultiArray.cs`.

When `AddFluidifyCompileItems` runs:

```xml
<Compile Remove="@(_FluidifyCompileOutput)" />
<Compile Include="@(_FluidifyCompileOutput)" />
```

The `Remove` with the absolute path doesn't match the relative SDK entry, so the file ends up in `Compile` twice.

--- a/src/Fluidify/build/Fluidify.targets
+++ b/src/Fluidify/build/Fluidify.targets
@@ -7,13 +7,13 @@
           Condition="'@(Fluidify)' != ''">
     <ItemGroup>
       <Fluidify Condition="'%(Fluidify.Destination)' == ''">
-        <OutputPath>%(RootDir)%(Directory)%(Filename)</OutputPath>
+        <OutputPath>%(RelativeDir)%(Filename)</OutputPath>
       </Fluidify>
       <Fluidify Condition="'%(Fluidify.Destination)' != '' and $([System.IO.Path]::IsPathRooted('%(Fluidify.Destination)'))">
         <OutputPath>%(Fluidify.Destination)</OutputPath>
       </Fluidify>
       <Fluidify Condition="'%(Fluidify.Destination)' != '' and !$([System.IO.Path]::IsPathRooted('%(Fluidify.Destination)'))">
-        <OutputPath>$(MSBuildProjectDirectory)\%(Fluidify.Destination)</OutputPath>
+        <OutputPath>%(Fluidify.Destination)</OutputPath>
       </Fluidify>
     </ItemGroup>
   </Target>

--- a/tests/Fluidify.Tests.Functional/CompileTests.cs
+++ b/tests/Fluidify.Tests.Functional/CompileTests.cs
@@ -72,6 +72,24 @@ public class CompileTests
             "Expected a missing type/namespace compiler error");
     }
 
+    [Test]
+    public async Task RebuildWithExistingOutput_NoDuplicateCompileWarning()
+    {
+        string compileAppProject = Path.Combine(CompileAppDir, "CompileApp.csproj");
+
+        // First build: generates the .cs file on disk
+        (int ExitCode, string Output) firstBuild = await RunDotnet($"build \"{compileAppProject}\" --force");
+        Assert.That(firstBuild.ExitCode, Is.EqualTo(0),
+            $"First build failed:\n{firstBuild.Output}");
+
+        // Second build: SDK glob picks up the file AND Fluidify adds it — should not duplicate
+        (int ExitCode, string Output) secondBuild = await RunDotnet($"build \"{compileAppProject}\" --force");
+        Assert.That(secondBuild.ExitCode, Is.EqualTo(0),
+            $"Second build failed:\n{secondBuild.Output}");
+        Assert.That(secondBuild.Output, Does.Not.Contain("CS2002"),
+            $"Duplicate compile warning detected on rebuild:\n{secondBuild.Output}");
+    }
+
     private static async Task<(int ExitCode, string Output)> RunDotnet(string arguments)
     {
         ProcessStartInfo psi = new ProcessStartInfo("dotnet", arguments)

